### PR TITLE
✨Model Changes

### PR DESCRIPTION
**Pull request by parimalak**
- [x] Add short_name to study
 Add external_id to every entity that does not already have it
- [x]  investigator, 
- [x] study_file, 
- [x] outcome, 
- [x] phenotype, 
- [x] family_relationship, 
- [x] sequencing_center, 
- [x] genomic_file,
- [x]  workflow
- [x] Add ncit_id field to diagnosis, biospecimen( one for tissue_type and one for tumor_descriptor)
- [x] availability field to genomic_file
Add source_text_ to
- [x]  phenotype  in phenotype
- [x] diagnosis, tumor_location in diagnosis
- [x] anatomical_site in biospecimen
- [x] Add spatial_descriptor to biospecimen and diagnosis

Addresses #231, #237 , #236, #232, #246, #249

### CODE REVIEW

File: migrations/versions/1cbf860fdd03_.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
+"""empty message
```
Comment (znatty22) [inline migrations/versions/1cbf860fdd03_.py]: Update docstring

**Review by znatty22 — COMMENTED** [review]
Already talked about this IRL, but so we remember:
- Combine migrations into 1
- Make <ontology>_id  fields consistently named

**Review by znatty22 — APPROVED** [review]
😎

File: migrations/versions/d9ba1b15d196_.py [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+    op.drop_column('biospecimen', 'anatomical_site')
+    op.add_column('diagnosis', sa.Column('icd_id_diagnosis', sa.Text(), nullable=True))
+    op.add_column('diagnosis', sa.Column('mondo_id_diagnosis', sa.Text(), nullable=True))
+    op.add_column('diagnosis', sa.Column('ncit_id_diagnosis', sa.Text(), nullable=True))
+    op.add_column('diagnosis', sa.Column('source_text_diagnosis', sa.Text(), nullable=True))
+    op.add_column('diagnosis', sa.Column('source_text_tumor_location', sa.Text(), nullable=True))
+    op.add_column('diagnosis', sa.Column('spatial_descriptor', sa.Text(), nullable=True))
+    op.add_column('diagnosis', sa.Column('uberon_id_tumor_location', sa.Text(), nullable=True))
+    op.drop_column('diagnosis', 'tumor_location')
+    op.drop_column('diagnosis', 'diagnosis')
```
Comment (dankolbman) [inline migrations/versions/d9ba1b15d196_.py:41]: For the future, we will want to use `alter_column` instead of `drop_column` to preserve the data.

**Review by dankolbman — APPROVED** [review]
👍

**Pull request merged**